docs/how-to-build-a-lexical-model.md

```diff
@@ -0,0 +1,365 @@
+---
+Author: Eddie Antonio Santos
+Date: July 22, 2019
+Keyman Version: 12.0.7-alpha-e58173f
+---
+
+How to build a lexical model
+============================
+
+A **lexical model** or a **dictionary** is a way to enable **predictive text**
+for your keyboard.
+
+Requirements
+------------
+
+You will need to have:
+
+ - [Node.js][] >= 10.0
+ - (Windows only) [Git Bash](https://gitforwindows.org/)
+
+
+[Node.js]: https://nodejs.org/en/
```

Comment:
Might as well put that link inline if you doing that for Git Bash anyway (but formatting will change when it moves...)